sdk/cosmos/azure-cosmos/requirements.txt

```diff
@@ -1,2 +1,2 @@
-azure-core<2.0.0,>=1.0.0b4
+azure-core<2.0.0,>=1.0.0
```

Comment:
Why this file exists in the first place? Can we try to delete it?

Reply:
Seems reasonable, everything in here is already in `shared_requirements.txt`